letsencrypt-win-simple/Plugin/ManualPlugin.cs

```diff
@@ -108,31 +108,29 @@ public override void HandleMenuResponse(string response, List<Target> targets)
                     Console.Write("Enter all Alternative Names seperated by a comma ");
                     Console.SetIn(new System.IO.StreamReader(Console.OpenStandardInput(8192)));
                     var sanInput = Console.ReadLine();
-                    alternativeNames = sanInput.Split(',');
+                    if (sanInput != null) alternativeNames = sanInput.Split(',');
```

Comment:
This section is only ran if a user specifies to run in --san. Therefore no null checks should happen in here.